build.sbt

```diff
@@ -55,47 +53,23 @@ val modelTests =
       )
     )
 
-val templates =
-  project
-    .in(file("templates"))
-    .enablePlugins(NoPublishPlugin)
-    .dependsOn(model.jvm)
-    .settings(
-      libraryDependencies ++= Seq(
-        "edu.gemini" %% "clue-core"   % clueVersion,
-        "edu.gemini" %% "lucuma-core" % lucumaCoreVersion
-      )
-    )
-
 val lucumaSchemas =
   crossProject(JVMPlatform, JSPlatform)
     .crossType(CrossType.Pure)
     .in(file("lucuma-schemas"))
     .dependsOn(model)
     .settings(
-      moduleName := "lucuma-schemas",
+      moduleName                    := "lucuma-schemas",
       libraryDependencies ++= Seq(
-        "edu.gemini"    %%% "clue-core"           % clueVersion,
         "co.fs2"        %%% "fs2-io"              % fs2Version             % Test,
         "org.scalameta" %%% "munit"               % munitVersion           % Test,
         "org.typelevel" %%% "munit-cats-effect-3" % munitCatsEffectVersion % Test
       ),
-      Compile / sourceGenerators += Def.taskDyn {
-        val root    = (ThisBuild / baseDirectory).value.toURI.toString
-        val from    = (templates / Compile / sourceDirectory).value
-        val to      = (Compile / sourceManaged).value
-        val outFrom = from.toURI.toString.stripSuffix("/").stripPrefix(root)
-        val outTo   = to.toURI.toString.stripSuffix("/").stripPrefix(root)
-        Def.task {
-          (templates / Compile / scalafix)
-            .toTask(s" GraphQLGen --out-from=$outFrom --out-to=$outTo")
-            .value
-          (to ** "*.scala").get
-        }
-      }.taskValue,
-      // Include schema files from templates in jar.
-      Compile / unmanagedResourceDirectories += (templates / Compile / resourceDirectory).value
+      Compile / clueSourceDirectory := (ThisBuild / baseDirectory).value / "lucuma-schemas" / "src" / "clue",
+      // Include schema files in jar.
+      Compile / unmanagedResourceDirectories += (Compile / clueSourceDirectory).value / "resources"
```

Comment:
Ah, I guess it's just to reference it here. Can we just put the schema file somewhere else?

Reply:
Not sure when this is used. The schema files are read by the generator from the directories specified in `scalafix.conf` in the property `GraphQLGen.schemaDirs`.

Reply:
Oh, I thought that downstreams needed this, so they can read it out of the resources in the Jar. is that not the case?

Reply:
Ah, yes, sorry. It has to be included in the Jar indeed. And yes, it makes sense to automatically include it. Do you think it's possible to automatically add it when invoking `scalafix GraphQLGen` so that users don't have to manually add it to `scalafix.conf`?

Reply:
> Do you think it's possible to automatically add it when invoking

Oh yeah, that would be nice as well. Not sure if we can pass arbitrary config when invoking scalafix but maybe we can backchannel it somehow 🤔